### PR TITLE
Updated test; added interfaces; completed code coverage

### PR DESCRIPTION
**Pull request by MikeTschudi**

### CODE REVIEW

File: packages/common/src/arcgisRestJS.ts [hunk 1]
```diff
@@ -53,19 +54,17 @@ import {
   shareItemWithGroup as restShareItemWithGroup,
   unprotectGroup as restUnprotectGroup,
   unprotectItem as restUnprotectItem,
+  updateGroup,
```
[the remark this answers is not in the document]
Comment (previnWong) [reply packages/common/src/arcgisRestJS.ts:57]: sorry, my bad, this was an import.

**Review by previnWong — APPROVED** [review]

**Comment by previnWong** [comment]
@MikeTschudi , didn't realize it till after I merged.  But this is referencing the direct modules.  I changed the 2 here to look at arcgisRestJS instead.  No errors in test scripts. but code coverage is less than 100% now.

image

image

**Comment by MikeTschudi** [comment]
@previnWong, since arcgisRestJS.ts references arcgis-rest-js, wouldn't it be reasonable for its test code to also reference that repo?

**Comment by previnWong** [comment]
I figured it's testing it indirectly.  But I guess you are right.  Direct with arcGIS Rest JS makes the most sense if there is one place.

**Pull request merged**